pyairtable/orm/model.py

```diff
@@ -171,6 +187,8 @@ def _get_meta(cls, name: str, default: Any = None, required: bool = False) -> An
         if required and not hasattr(cls.Meta, name):
             raise ValueError(f"{cls.__name__}.Meta.{name} must be defined")
         value = getattr(cls.Meta, name, default)
+        if callable(value):
```

Comment:
Beyond a nit and this particular usage is already very concise so feel free to completely disregard, PR looks great but was poking around in Python 3.10 docs and this seems like a case where match (ref [PEP 634/635/636](https://peps.python.org/pep-0636/) could be used in a world where 3.10 is the minimum supported version ie:

```
match value:
    case callable_func := callable(value):
        value = callable_func()
```

Reply:
I will admit to not having my head wrapped around the `match` syntax (or [soft keywords](https://docs.python.org/3/reference/lexical_analysis.html#soft-keywords) in general) but I think it's fair to say we shouldn't use it if it's not present (or available via `__future__`) in all supported Python versions.